deployment/config.go

```diff
@@ -170,6 +170,8 @@ type TerraformDBSettings struct {
 	// If ClusterIdentifier is not empty, DBName should be set to the name of the database in such cluster.
 	// If ClusterIdentifier is empty, the database created will use DBName as its name.
 	DBName string `default:""`
+	// EngineVersion specifies the version of the database engine.
+	EngineVersion string `default:""`
```

Comment:
Have a default value here.

And also, I'd like to have some validation where the engine version is matched to a set of valid version strings depending on the engine type. AWS will automatically throw an error if there's a mistake, so it's better to do some checking from our side.

Reply:
Should I go ahead and default targeting postgres only?

Reply:
I think that should be fine.

Reply:
So, thinking about this a bit more. If I set up a default value here the actual defaults are not going to be used:

```tf
  engine_version      = var.db_engine_version != "" ? var.db_engine_version : var.db_default_engine_version[var.db_instance_engine]
```